src/modules/dashboard/components/HookedTokenIcon/HookedTokenIcon.tsx

```diff
@@ -44,11 +45,21 @@ const loadTokenImages = async (address: Address): Promise<Response> => {
   return fetch(tokenImageUrl);
 };
 
+const getBase64image = (blob): Promise<any> => {
+  return new Promise((resolve) => {
+    const fileReader = new FileReader();
+    fileReader.onload = function () {
+      return resolve(fileReader.result);
+    };
+    fileReader.readAsDataURL(blob);
+  });
+};
+
 const HookedTokenIcon = ({
   name,
   token: { iconHash, address },
   iconName,
-  dontFetch,
+  dontFetch = process.env.NODE_ENV === 'development',
```

Comment:
This should actually go by network, since that repo only has icon for mainnet ethereum

So something like:
```
DEFAULT_NETWORK !== Network.Mainnet
```